scipy/linalg/_decomp_update.pyx.in

```diff
@@ -725,9 +724,15 @@ cdef int qr_block_col_insert(int m, int n, blas_t* q, int* qs,
         # First zero the lower part of the new columns using a qr.
 
         # query the workspace,
+        # set tau to point at something to keep new MKL working.
+        tau = &c
         info = geqrf(m-n+p, p, index2(r, rs, n-p, k), m, tau, &c, -1)
+        if info < 0:
+            return libc.stdlib.abs(info)
```

Comment:
It is so that the code that prints the error message knows if the error was in geqrf or ormqr.  See line 2016.